actions/debloat-runner/action.yaml

```diff
@@ -0,0 +1,13 @@
+name: "Debloat runner"
+description: "Remove unnessecary libraries & binaries for more leeway on space intensive builds"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository code
+      uses: actions/checkout@v4
```

Comment:
Is checking out here necessary? I think it might even overwrite some authorization towards GitHub in the worst case since `persist-credentials: false` is not set.